src/vuestic-theme/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -171,7 +175,11 @@ export default {
     initPopper () {
       const options = {
         placement: this.position || 'bottom',
-        modifiers: {},
+        modifiers: {
+          preventOverflow: {
+            enabled: this.preventOverflow,
```

Comment:
This might be a hack from my side. You can comment this out then see if dropdowns become more fixed.